packages/dds/tree/src/feature-libraries/object-forest/objectForest.ts

```diff
@@ -321,6 +377,16 @@ class Cursor implements ITreeSubscriptionCursor {
         return this.nodeStack[this.nodeStack.length - 1];
     }
 
+    getParentFieldKey(): FieldKey | DetachedField {
+        assert(this.state === ITreeSubscriptionCursorState.Current, 0x33d /* Cursor must be current to be used */);
```

Comment:
The build is complaining about this error code being copied from line 375.

Reply:
fixed